core/chain_makers.go

```diff
@@ -61,7 +62,7 @@ func (b *BlockGen) SetCoinbase(addr common.Address) {
 		panic("coinbase can only be set once")
 	}
 	b.header.Coinbase = addr
-	b.gasPool = new(GasPool).AddGas(CalcGasLimit(b.parent, b.statedb)) //TODO: fix this
+	b.gasPool = new(GasPool).AddGas(params.DefaultGasLimit)
```

Comment:
I guess this only used in tests and so it doesn't really matter whether it uses the contract or the default?  I guess here it could be possible to use some kind of testing-only stub struct which satisfies VMRunner.  Though I'm guessing here it doesn't matter so probably not worth it.

Still, a couple of questions:
1. What did that `//TODO: fix this` mean?  I don't see anything obviously wrong with the old code.  does it have to do with the parent and state not being set?
2. If we're going to just use the default gas limit, update the comment on this function to remove Note since it'd no longer be applicable.